### PR TITLE
willys - v2 CI/CD blueprints

### PR DESCRIPTION
**Pull request by wpschaub**
Focus is on classic versus YAML vs v2 blueprints.

### CODE REVIEW

File: content/back-to-basics-why-are-we-moving-to-v2-blueprints.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+We covered our move from classic to YAML-based Azure Pipelines in a detailed series, which begins at [Part 1: Pipelines - Why bother and what are our nightmares and options?](/yaml-pipelines-part1.html) and  triggered in our open-source initiative as covered in posts such as [Our road to OSS Blueprints - Suppress CD when pipeline runs within Pull Request](https://wsbctechnicalblog.github.io/yaml-pipelines-part11.html), [Azure Pipelines Blueprint QA Integration](https://wsbctechnicalblog.github.io/azure-pipelines-blueprint-qa-integration.html) and [Azure DevOps Pipeline OSS v2.1 Flow](https://wsbctechnicalblog.github.io/azure-devops-pipeline-oss-v2-1-flow.html). 
+
+In this post, I will only answer the question of **WHY** and let you peruse our other posts for the implementation details.
+
+---
+
+# WHY are we moving away from and disabling classic Azure Pipelines?
+
+The following image previous shows that **Classic** pipelines are the new name for the pipelines that started with MSBuild in Team Foundation Server (TFS) 2005, developed to XAML-based pipelines in TFS 2010, and finally to visual JSON-based pipelines in TFS 2015.
```
Comment (MerlinDarkstar) [inline content/back-to-basics-why-are-we-moving-to-v2-blueprints.md]: extra word - "previous" (4th word)

File: content/back-to-basics-why-are-we-moving-to-v2-blueprints.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+> ![Alert](/images/back-to-basics-batch-size-alert.png)
+>
+> By the end of this year, our classic pipelines will be locked, so teams can still use them, but not modify them or make new ones.
+>
+
+---
+
+# WHY are we moving to YAML-based Azure Pipelines?
+
+The YAML pipeline introduced an exciting new era, launching in early 2019 with TFS 2019. It signaled a CI/CD solution designed from scratch on aN INNOVATIVE whiteboard.
```
Comment (MerlinDarkstar) [inline content/back-to-basics-why-are-we-moving-to-v2-blueprints.md]: excess capitalization "aN INNOVATIVE" - s/b "an INNOVATIVE"

**Review by MerlinDarkstar — DISMISSED** [review]
Looks great Willy!

**Review by littleswancode — DISMISSED** [review]

**Review by danielrbroderick — DISMISSED** [review]

File: content/back-to-basics-why-are-we-moving-to-v2-blueprints.md (hunk 1 above)
Comment (wpschaub) [reply content/back-to-basics-why-are-we-moving-to-v2-blueprints.md]: Fixed.

File: content/back-to-basics-why-are-we-moving-to-v2-blueprints.md (hunk 2 above)
Comment (wpschaub) [reply content/back-to-basics-why-are-we-moving-to-v2-blueprints.md]: Fixed.

**Pull request merged**